controllers/cluster-registration/cluster_registration_controller.go

```diff
@@ -309,13 +332,102 @@ func (r *RegisteredClusterReconciler) syncManagedServiceAccount(regCluster *sing
 	return nil
 }
 
+func (r *RegisteredClusterReconciler) processRegclusterDeletion(regCluster *singaporev1alpha1.RegisteredCluster, managedCluster *clusterapiv1.ManagedCluster, hubCluster *helpers.HubInstance) (ctrl.Result, error) {
```

Comment:
Do we need to also delete the import command configmap and kubeconfig secret or are those automatically cleaned up? The kubeconfig secret has the owner set but it doesn't look like the configmap does.

Reply:
I am adding ownerref to configmap in my ClusterID status PR.